dss_datamover/utils/config.py

```diff
@@ -206,6 +206,12 @@ def put(self, subparser):
                                help='Profiling of PUT operation (Not Implemented)')
         subparser.add_argument("--standalone", "-sa", required=False, action='store_true',
                                help='Run in standalone mode')
+        subparser.add_argument("--nfs-server", "-ns", type=str, required='--nfs-port' in sys.argv or '--nfs-share' in sys.argv,
```

Comment:
required should be False. This is just an option to override the configuration parameters. Completely optional
Same for the other 2 parameters as well